apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/metrics/ScanMediatorVerdictIngestionMetrics.scala

```diff
@@ -6,21 +6,21 @@ package org.lfdecentralizedtrust.splice.scan.metrics
 import com.daml.metrics.api.MetricHandle.{Gauge, LabeledMetricsFactory, Meter}
 import com.daml.metrics.api.{MetricInfo, MetricName, MetricsContext}
 import com.daml.metrics.api.MetricQualification.Traffic
+import com.digitalasset.canton.data.CantonTimestamp
 import org.lfdecentralizedtrust.splice.environment.SpliceMetrics
 
 class ScanMediatorVerdictIngestionMetrics(metricsFactory: LabeledMetricsFactory)
     extends AutoCloseable {
   private val prefix: MetricName = SpliceMetrics.MetricsPrefix :+ "scan" :+ "verdict_ingestion"
 
-  type CantonTimestampMicros = Long
-
-  val lastIngestedRecordTime: Gauge[CantonTimestampMicros] = metricsFactory.gauge(
+  val lastIngestedRecordTime: Gauge[CantonTimestamp] = SpliceMetrics.cantonTimestampGauge(
+    metricsFactory,
     MetricInfo(
       name = prefix :+ "last_record_time_us",
       summary = "Latest ingested mediator verdict record_time",
       qualification = Traffic,
     ),
-    initial = 0L,
+    initial = CantonTimestamp.MinValue,
```

Comment:
Changing the initial value might mess up some existing alerts/dashboards, but all other timestamp gauges have `MinValue` as the initial value and this metric is new, so changing it makes sense.